docs/notebooks/wasserstein_barycenters_gmms.ipynb

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Initialize the means and the covariances of the GMM barycenter.
</pre><p>Maybe this could be promoted to a text cell rather than a comment line. Maybe worth explaining why we need to implement an init that makes sense (to account for psd constraint on covs)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>ok</p>